src/components/UserMenu/UserMenu.tsx

```diff
@@ -108,86 +130,86 @@ export class UserMenu extends React.Component<UserMenuProps, UserMenuState> {
+                    isClickable ? 'clickable' : ''
+                  }`}
                 >
-                  <div className="image">
-                    <AvatarFace size="small" avatar={avatar} />
-                  </div>
-                  <div>
-                    <div className="name">{name || i18n.guest}</div>
+                  <div
+                    className={`info ${onClickProfile ? 'clickable' : ''}`}
+                    onClick={onClickProfile}
+                  >
+                    <div className="image">
+                      <AvatarFace size="small" avatar={avatar} />
+                    </div>
+                    <div>
+                      <div className="name">{name || i18n.guest}</div>
+                    </div>
                   </div>
+                  <ul className="actions">
+                    <a href="https://account.decentraland.org">
+                      <li>
+                        <Icon name="user"></Icon>
+                        {i18n.account}
+                      </li>
+                    </a>
+                    {menuItems}
+                    {onClickSettings ? (
+                      <li onClick={onClickSettings}>
+                        <Icon name="cog"></Icon>
+                        {i18n.settings}
+                      </li>
+                    ) : null}
+                    {onSignOut ? (
+                      <li onClick={onSignOut}>
+                        <i className="sign-out-icon" />
+                        {i18n.signOut}
+                      </li>
+                    ) : null}
+                  </ul>
                 </div>
-                <ul className="actions">
-                  <a href="https://account.decentraland.org">
-                    <li>
-                      <Icon name="user"></Icon>
-                      {i18n.account}
-                    </li>
-                  </a>
-                  {menuItems}
-                  {onClickSettings ? (
-                    <li onClick={onClickSettings}>
-                      <Icon name="cog"></Icon>
-                      {i18n.settings}
-                    </li>
-                  ) : null}
-                  {onSignOut ? (
-                    <li onClick={onSignOut}>
-                      <i className="sign-out-icon" />
-                      {i18n.signOut}
-                    </li>
-                  ) : null}
-                </ul>
-              </div>
-            </>
-          )}
-          {!isSignedIn && (
-            <Button primary disabled={isSigningIn} onClick={onSignIn}>
-              {i18n.signIn}
-            </Button>
-          )}
-        </div>
-      </Row>
+              </>
+            )}
+            {!isSignedIn && (
+              <Button primary disabled={isSigningIn} onClick={onSignIn}>
+                {i18n.signIn}
+              </Button>
+            )}
+          </div>
+        </Row>
+        {isSignedIn && isSomeBalanceTooHigh && (
```

Comment:
Not sure I followed this code here, if some balances are too high, for mobile we render the balance again? (now wrapped with `mobile-user-balances-wrapper`) but it's already rendered [here](https://github.com/decentraland/ui/pull/348/files#diff-945e43fecdb33b773f7feb482a363ec6c8d7ac6737c18a71ad8064a31ca4e260R154)?

Reply:
Thanks for the review @juanmahidalgo! When the user balance is too high, the balances next to the user avatar are hidden using this CSS: https://github.com/decentraland/ui/pull/348/files#diff-d2f8fdb7b3d34c51e2ee2c2a1470b3a4488ef6e865da0dc703e482455b64ec19R206-R210

that way, it shows only one balance at a time